libavcodec/vvc/vvc_ps.c

```diff
@@ -1012,6 +1029,43 @@ static int sps_parse_vui(VVCSPS *sps, GetBitContext *gb, void *log_ctx)
     return 0;
 }
 
+static int sps_parse_extension(VVCSPS *sps, GetBitContext *gb, void *log_ctx)
+{
+    int range_extension_flag;
+    sps->extended_precision_flag = 0;
+    sps->ts_residual_coding_rice_present_in_sh_flag = 0;
+    sps->rrc_rice_extension_flag = 0;
+    sps->persistent_rice_adaptation_enabled_flag = 0;
+    sps->reverse_last_sig_coeff_enabled_flag = 0;
+
+    if (get_bits1(gb)) {  // sps_extension_present_flag
+        range_extension_flag = get_bits1(gb);
```

Comment:
better write to "const int range_extension_flag", this will tell reader, the range_extension_flag only used in the if